packages/storybook/.stylelintrc.json

```diff
@@ -1,6 +1,6 @@
 {
   "extends": ["stylelint-config-standard-scss"],
   "rules": {
-    "selector-class-pattern": "^(dt-[a-z]+(--[a-z]+(-[a-z])+)?|docs-story|visual-regression-wrapper|section)"
+    "selector-class-pattern": "^(dt-[a-z]+(--[a-z]+(-[a-z])+)?|docs-story|visual-regression-wrapper|section|citation|story-grid)"
```

Comment:
Het voelt niet alsof dit prefixes zijn, en dat is waar de lint regel voor bedoeld is
Is het niet handiger om te kiezen uit de volgende opties:

a) een prefix gebruiken voor je class names, zoals `docs`
b) `packages/storybook` in `.stylelintignore` zetten